src/server/status/metrics.js

```diff
@@ -23,5 +25,21 @@ module.exports = function (kbnServer, server, config) {
       requestsPerSecond: requestsPerSecond
     });
 
+    // Creates a status object for this node
+    kbnServer.stats = {
+      load: event.osload,
```

Comment:
As quickly discussed, this is based on the metricbeat naming conventions: https://www.elastic.co/guide/en/beats/libbeat/5.1/event-conventions.html